Problem: upgrade to webpki 0.21.4 fails to compile

Possibly due to the custom ring fork for sgx
```
error[E0425]: cannot find function `fill_impl` in this scope
225	   --> /drone/src/drone/cargo/registry/src/github.com-1ecc6299db9ec823/ring-0.16.19/src/rand.rs:160:9
226	    |
227	160 |         fill_impl(dest)
228	    |         ^^^^^^^^^ help: try: `self.fill_impl`
229	
230	   Compiling thread-pool v0.1.1
231	error: aborting due to previous error
232	
233	For more information about this error, try `rustc --explain E0425`.
234	error: could not compile `ring`.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -55,7 +55,7 @@ default-members = [
 ]
 
 [patch.crates-io]
-ring = { git = "https://github.com/crypto-com/ring.git", branch = "sgx-env" }
+ring = { git = "https://github.com/crypto-com/ring.git", rev = "493eb07f3a7118870867093bf934f1e37a1aa602" }
 # FIXME: use upstream when merged
 sha2 = { git = "https://github.com/crypto-com/hashes.git", rev = "289d5b76f2163a3808010341ed1df3cb156d97e1" }
 # FIXME: before official spec has a solution

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -55,7 +55,7 @@ default-members = [
 ]
 
 [patch.crates-io]
-ring = { git = "https://github.com/crypto-com/ring.git", rev = "8f2b68d2ac53b1df61ca5cdc1190f86d74e302a3" }
+ring = { git = "https://github.com/crypto-com/ring.git", branch = "sgx-env" }
 # FIXME: use upstream when merged
 sha2 = { git = "https://github.com/crypto-com/hashes.git", rev = "289d5b76f2163a3808010341ed1df3cb156d97e1" }
 # FIXME: before official spec has a solution

